src/components/ArmyBuildPane.jsx

```diff
@@ -0,0 +1,23 @@
+
+import React from 'react';
+import styled from 'styled-components'
+
+import ArmyList from'./ArmyList';
+import SelectionPane from './SelectionPane'
+
+const Container = styled.div`
+    display: flex;
+    height: 70vh;
+    margin: 30px
+`;
+
+const ArmyBuildPane = (props) => {
+    return (
+        <Container>
+            <SelectionPane/>
+            <ArmyList/>
+        </Container>
+    );
+};
```

Comment:
More implicit returning (also you're not using any props so I guess you don't need them?):
```suggestion
const ArmyBuildPane = () => (
    <Container>
        <SelectionPane />
        <ArmyList />
    </Container>
);
```